contracts/callpaths/WarmPath.sol

```diff
@@ -87,7 +87,26 @@ contract WarmPath is MarketSequencer, SettleLayer, ProtocolAccount {
                        uint128 limitLower, uint128 limitHigher,
                        address lpConduit)
         private returns (int128, int128) {
-        if (code == UserCmd.MINT_AMBIENT_LIQ_LP) {
+        if (code == UserCmd.MINT_RANGE_LIQ_LP) {
+            return mintConcentratedLiq(base, quote, poolIdx, bidTick, askTick, liq, lpConduit,
```

Comment:
can we remove the bidTick and askTicks from the entrypoints and refer to a pool template to gather the ticks we want

Reply:
i don't think we store the ticks on the pool template also this the the exact logic which crocswap protocol use to mint concentrated liq

Reply:
would be a better solution to define ranges on a pool template rather than hard coding them so we can change them later

Reply:
if i understand correctly, user wont have ticks option for providing liquidity, it will always be from x to y, not anything in between x and y.

Reply:
Discussed with @ocnc, we should keep the ticks in the input to allow user to provide liquidity in between the set ticks